src/conversions.rs

```diff
@@ -133,6 +126,19 @@ pub fn multi_addr(ip_address: Ipv4Addr, port: u16) -> Multiaddr {
     format!("/ip4/{}/tcp/{}", ip_address, port).parse().unwrap()
 }
 
+/// Check the distance between 2 peers. Subscribe to new peer's individual topic
+/// if the distance is below [MAX_REPLICA_DISTANCE]
+pub(crate) fn is_close_peer(peer_1: &PeerId, peer_2: &PeerId) -> bool {
```

Comment:
Any reason why you think this function should be in the "conversions" module? In what way is it a conversion?

Reply:
Will shift this to peer module.